src/index.scss

```diff
@@ -274,12 +289,25 @@ h3 {
       }
     }
 
-    .record-count > span {
-      min-width: 100px;
-      display: flex;
+    .record-count {
+      justify-self: end;
+      min-width: 150px;
+
+      span {
+        min-width: 150px;
+        display: flex;
+      }
     }
   }
 
+  div {
```

Comment:
I think something went wrong with your rebasing point? I'm seeing changes that already in master now? In the future, no need to rebase/squash before merging. GitHub has as "squash and merge" feature built in.

Reply:
Github was giving me a merge conflict, so I pulled from dev and resolved them on my end. I probably read what Justin said wrong though and rebased unnecessarily.